lista03/ex01/ex01/src/ex01/Automovel.java

```diff
@@ -0,0 +1,26 @@
+package ex01;
+
+public class Automovel extends Veiculo {
+    protected String placa;
+    
+    Automovel (float precoAutomovel, String modeloAutomovel, String marcaAutomovel, String corAutomovel, String placaAutomovel) {
+        super(precoAutomovel, modeloAutomovel, marcaAutomovel, corAutomovel);
+        placa = placaAutomovel;
+    }
+    
+    void listarVerificacoes() {
+        System.out.println("Verificacoes");
+    }
+    
+    void ajustar() {
+        System.out.println("Ajustado");
+    }
+    
+    void limpar() {
+        System.out.println("Limpado");
```

Comment:
Diz-se que o veículo foi limpo
```suggestion
        System.out.println("Limpo");
```